All_materials/20230822_Web_Scraping_Financial_Ombudsman_Service/README.md

```diff
@@ -0,0 +1,82 @@
+Web Scraping Complaints from the Financial Ombudsman Service
+=============================
+
+#### *Kenneth Quan*
+<kenneth.quan1@nhs.net>
```

Comment:
do you want to put your email address out there?

Reply:
I removed the email